### PR TITLE
DOC: GH42204: Change docstring for head and tail in core/generic.py

### LINKED ISSUE #42204: Absolute value or minus sign missing in docstrings for head and tail
In the docstrings for the head and tail functions (in pandas/core/generic.py) there is an absolute value or minus sign missing:
e.g. in tail docstring
```
For negative values of `n`, this function returns all rows except
        the first `n` rows, equivalent to ``df[n:]``.
```

should read something like
```
For negative values of `n`, this function returns all rows except
        the first `np.abs(n)` rows, equivalent to ``df[np.abs(n):]``.
```

or 

```
For negative values of `n`, this function returns all rows except
        the first `-n` rows, equivalent to ``df[-n:]``.
```
Similarly for head.

### PR DESCRIPTION
**Pull request by olbermann**
- [X] closes #42204
- [X] As test, ran scripts/validate_docstring.py with outputs
Docstring for "pandas.core.generic.NDFrame.head" correct. :)
Docstring for "pandas.core.generic.NDFrame.tail" correct. :)

### CODE REVIEW

File: pandas/core/generic.py [hunk 1]
```diff
@@ -5001,8 +5001,8 @@ def head(self: FrameOrSeries, n: int = 5) -> FrameOrSeries:
         on position. It is useful for quickly testing if your object
         has the right type of data in it.
 
-        For negative values of `n`, this function returns all rows except
-        the last `n` rows, equivalent to ``df[:-n]``.
+        For a positive integer `k`, the function ``df.head(-k)`` returns all rows
```
Comment (mzeitlin11) [inline pandas/core/generic.py:5004]: Nit: the word function doesn't make as much sense in this usage

File: pandas/core/generic.py [hunk 2]
```diff
@@ -5074,8 +5074,8 @@ def tail(self: FrameOrSeries, n: int = 5) -> FrameOrSeries:
         position. It is useful for quickly verifying data, for example,
         after sorting or appending rows.
 
-        For negative values of `n`, this function returns all rows except
-        the first `n` rows, equivalent to ``df[n:]``.
+        For a positive integer `k`, the function ``df.tail(-k)`` returns all rows
```
Comment (mzeitlin11) [inline pandas/core/generic.py:5077]: Same as above for function

**Review by mzeitlin11 — COMMENTED** [review]
Thanks for the pr @olbermann! Some small initial comments, plus I think it would also be helpful to adjust the wording above the examples to match your changes here.  See here for example: https://github.com/pandas-dev/pandas/blob/d61ace50ee2329cecdd6f3f6d2c5254224ccd674/pandas/core/generic.py#L5055

**Comment by olbermann** [comment]
Thanks for spotting the wrong usage of "function". For the examples I think it would be nicer to keep things as they are.
So what do you think about a different suggestion:

in head docstring:

For a negative integer `n = -k`, this function returns all rows except
the last `k` rows, equivalent to ``df[:-k]``.

in tail docstring:

For a negative integer `n = -k`, this function returns all rows except
the first `k` rows, equivalent to ``df[k:]``.

and no changes above the examples?

(If yes, would it be best to do this in a new pr?)

**Comment by mzeitlin11** [comment]
I find that new wording harder to understand than your original, but curious if others have any thoughts (regardless, you can always just push new commits to this pr instead of opening a new one)

**Comment by rhshadrach** [comment]
+1 one on preferring the original wording (once modified based on existing feedback). Not sure it's necessary to adjust the examples - they use `n` (and that `n = -3` seems clear to me) instead of `k` so I don't find them confusing.

**Comment by attack68** [comment]
To be honest I find the original wording (current master) clearer, except for needing to insert `|n|` instead of `n` in one place. Albeit the meaning is obvious.

However, if these new changes are to be implemented I would suggest using the current implementation:

 - change "the function ``df.head(-k)``" to just "``df.head(-k)``"
 - change `k` back to `n`.

**Comment by mroeschke** [comment]
Thanks for the PR, but seems like the original wording with some modifications is preferable to what is here currently. Additionally, looks like this PR has gone stale. Going to close, but please ping if you're still interested in working on this and we can reopen.

**Pull request closed without merging**